test/tests/zookeeper-basics/run.sh

```diff
@@ -23,5 +23,5 @@ zkCli() {
 
 . "$dir/../../retry.sh" --tries "$ZOOKEEPER_TEST_TRIES" --sleep "$ZOOKEEPER_TEST_SLEEP" zkCli ls /
 
-# List Zokeeper root
-[ "$(zkCli ls / | tail -n1)" = '[zookeeper]' ]
+# List and check Zookeeper root
+[ ! -z "$(zkCli ls / | grep -o '\[zookeeper\]')" ]
```

Comment:
The following is the same intent as what this is testing, right? (just hopefully simpler and thus more reliable)

```suggestion
zkCli ls / | grep -qF '[zookeeper]'
```

Reply:
I suppose it would probably be better testing for this to be an exact line in the output instead, something like:

```suggestion
zkCli ls / | grep -q '^\[zookeeper\]$'
```

Reply:
Great idea! I went with `zkCli ls / | grep -q '^\[zookeeper\]'` because `^\[zookeeper\]$` doesn't work for versions other than `3.6`. I guess there's no line break.